streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -199,8 +199,11 @@ public long approximateNumEntries() {
 
     @Override
     public void close() {
-        super.close();
-        streamsMetrics.removeAllStoreLevelSensors(threadId, taskId, name());
+        try {
+            wrapped().close();
+        } finally {
+            streamsMetrics.removeAllStoreLevelSensors(threadId, taskId, name());
+        }
```

Comment:
This is a related bug I fixed. It's basically the same thing, but for Metered store. There's no need to use the new util when we're just dealing with one potentially-throwing operation.